src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -288,7 +288,7 @@ static bool hasMetadata(std::string name, std::string type, ContextAttribute* ca
 
     /* Note that, unlike entity types or attribute types, for attribute metadata we don't consider empty type
```

Comment:
After changing the if just below, this comment doesnt make sense any more. So, it shouls be removed.